torchci/pages/benchmark/compilers.tsx

```diff
@@ -210,7 +210,10 @@ function computePassrate(
       passCount[bucket][workflowId][suite][compiler] = 0;
     }
 
-    if (isPass(bucket, workflowId, suite, compiler, model, passingModels)) {
+    if (
+      isPass(bucket, workflowId, suite, compiler, model, passingModels) &&
+      record.speedup !== 0.0
```

Comment:
Is there never a legitimate scenario where the speedup is zero?

Reply:
It practically never happens, even when we run the same model twice. We could check for absolute execution time instead which may be a better indicator. I don't think it's worth adding an additional state to indicate whether or not the perf test errored out

Reply:
Makes sense, though I'd suggest adding a comment explaining why we're ignoring the zero case and the shortcut being taken